yt/visualization/plot_window.py

```diff
@@ -228,6 +229,14 @@ def __init__(
                 self._field_transform[field] = log_transform
             else:
                 self._field_transform[field] = linear_transform
+
+            log, linthresh = self._log_config[field]
+            if (log is not None and log is not finfo.take_log) or linthresh:
```

Comment:
Maybe I need more ☕ but I'm a little confused by the part after the `and` here. Is it necessary? Should `finfo.take_log` matter if the user is overriding it?

Reply:
You're right, the logic is broken here. I have fixed it.